go/private/context.bzl

```diff
@@ -73,6 +73,12 @@ load(
     "request_nogo_transition",
 )
 
+# cgo requires a gcc/clang style compiler.
+_UNSUPPORTED_C_COMPILERS = {
```

Comment:
I would probably go the other direction and enumerate that `gcc` and possibly `musl-gcc` are supported. It's unlikely that `tinycc` (or any of the other thousands of esoteric compilers) is a supported compiler but it's not on this list.

Reply:
I agree that would be conceptually more correct, but Bazel hasn't always been doing a good job at returning something meaningful in the `compiler` field: Until I believe 6.1.0, most compilers just have `"compiler"` in there. I don't think that there is any further unsupported compiler for which Bazel ever returned a distinctive compiler name.

Reply:
Could we leave a comment requesting the next person who edits this list update it to be a inclusion list instead of an exclusion list assuming version 6.1 to spend released and is generally in use by that point?